base/docs/basedocs.jl

```diff
@@ -1492,4 +1492,8 @@ See the manual section on [Tuple Types](@ref).
 """
 Tuple
 
+@doc """
+The base library of Julia.
+""" -> kw"Base"
```

Comment:
This can be:
```julia
"""
The base library of Julia.
"""
Base
```
instead of documenting as a keyword.

Reply:
Are you sure? I tried 
```Julia
@doc "..." -> Base
```
and `?Base` did not work.

Reply:
I just tested your suggestion, it does not work either.

Reply:
Hmm... I swear it worked yesterday when I tried :)